META.json

```diff
@@ -58,10 +58,11 @@
          "web" : "https://github.com/regentmarkets/duo_perl"
       }
    },
-   "version" : "0.01",
+   "version" : "0.02",
    "x_contributors" : [
       "Ben Murray <ben@duosecurity.com>",
       "Fayland Lam <fayland@gmail.com>",
+      "Raunak Kathuria <raunakkathuria@gmail.com",
       "Jon Oberheide <jon@oberheide.org>",
       "alichator <alishahc@umich.edu>"
    ]
```

Comment:
META.json should not be in the distribution. It is usually generated by `perl Makefile.PL`. The contributors list should also go to Makeflle.PL. Example - https://github.com/binary-com/perl-App-Base/blob/master/Makefile.PL